admin/src/env.d.ts

```diff
@@ -0,0 +1,20 @@
+interface ImportMeta {
+	readonly env: {
+		VITE_ADMIN_FB_AUTH_EMULATOR_URL: string;
```

Comment:
I like it 👍. 
Wondering if the config.tsx still adds some additional value or is just one more indirection?

Reply:
Very true. It's not really needed anymore. I'll clean it up.